deployment/build_and_release/live/prod/terragrunt.hcl

```diff
@@ -23,5 +23,22 @@ inputs = merge(
         cloudbuild_path = "release/cloudbuild.yaml"
       }
     }
+
+    build_substitutions = {
+      log_name = "armored-witness-firmware-log-prod-1"
```

Comment:
Just a thought: if it's not possible to use `key_version` and `env` directly here, one option could be to call this `log_name_prefix` (="armored-witness-firmware"), and then in the module build the final string with a fmt of `log_name_prefix`, `env`, and `key_version`?

Can do the same for firmware_bucket too, and maybe `origin`?

Reply:
Done.